CI/run_tests.sh

```diff
@@ -1,23 +1,36 @@
 #!/usr/bin/env bash
 
-sudo python ptf_nn/ptf_nn_agent.py \
+set -x
+
+THIS_SCRIPT_FILE_MAYBE_RELATIVE="$0"
+THIS_SCRIPT_DIR_MAYBE_RELATIVE="${THIS_SCRIPT_FILE_MAYBE_RELATIVE%/*}"
+THIS_SCRIPT_DIR_ABSOLUTE=`readlink -f "${THIS_SCRIPT_DIR_MAYBE_RELATIVE}"`
+
+MAJOR_MINOR_VERSION=`${THIS_SCRIPT_DIR_ABSOLUTE}/python-major-minor-version.py`
```

Comment:
Maybe a oneliner?
`MAJOR_MINOR_VERSION=$(python3 -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')`

Reply:
Replaced with one-liner in commit 23.